compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Passes.cpp

```diff
@@ -130,15 +133,15 @@ void addSimplePackBasedPassPipeline(OpPassManager &pm,
   pm.addPass(createCanonicalizerPass());
   pm.addPass(createCSEPass());
 
-  AMDAIETileAndFuseOptions tileOptions;
+  AMDAIETileAndFuseOptions tileFuseOptions;
   AMDAIEPackAndTransposeOptions packOptions;
   AMDAIEBufferizeToAllocationOptions bufferizeOptions;
 
   // First level tiling using scf.forall
-  tileOptions.tilingLevel = 0;
-  tileOptions.useSCFFor = false;
+  tileFuseOptions.tilingLevel = 0;
```

Comment:
Can be a foot gun to reuse the `tileFuseOptions`, etc.

Can we move them closer to the pass. Do something like
```
{
    AMDAIETileAndFuseOptions tileFuseOptions;
    ...
    modulePassManager.addNestedPass...(createAMDAIETileAndFusePass(tileFuseOptions));
}
```

Should be a simple change on top of this PR. Wont block on this, but please add that here or as a follow up.


```

Reply:
Done.